Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  FacilitiesDropdownCollectionViewCell.swift
+//  
+//
+//  Created by Cameron Hamidi on 11/6/19.
+//
+
+import UIKit
+
+class FacilitiesDropdownCell: UICollectionViewCell {
+
+    private enum FacilityName: String {
+        case bowlingLanes = "BOWLING LANES"
+        case equipment = "EQUIPMENT"
+        case fitnessCenter = "FITNESS CENTER"
+        case gymnasium = "GYNMASIUM"
+        case swimmingPool = "SWIMING POOL"
+    }
+
+    private let headerImageView = UIImageView()
+    private let headerNameLabel = UILabel()
+    private let headerOpenLabel = UILabel()
+    private let headerView = UIView()
+
+    private var collectionView: UICollectionView!
+    private var collectionViewHeight: CGFloat = 0
+    private var dropdownView: DropdownView!
+    private var facility: Facility!
+    private var height: CGFloat = 0
+
+    override init(frame: CGRect) {
+        let minimumInteritemSpacing: CGFloat = 16
+        super.init(frame: frame)
+
+        let collectionViewLayout = UICollectionViewFlowLayout()
+        collectionViewLayout.minimumInteritemSpacing = minimumInteritemSpacing
```

Comment:
I think we can just do `collectionViewLayout.minimumInteritemSpacing = 16` instead of making a variable